pleskdistup/main.py

```diff
@@ -194,8 +194,9 @@ def do_convert(
     show_plan: bool,
 ) -> int:
     if not options.resume and not required_conditions_satisfied(upgrader, options, options.phase):
-        printerr("Please fix noted problems before continuing the conversion")
-        return 1
+        printerr("Conversion can't be performed due to the problems noted above")
+        if not show_plan:
+            return 1
 
     actions_map = upgrader.construct_actions(sys.argv[0], options, options.phase)
     dup = find_duplicate_actions(actions_map)
```

Comment:
By the way, not sure we should avoid duplicates. I believe it is fine if we want to perform the same action several times. However it not tied to the changes itself.

Reply:
Here only actions with the same name inside the same stage are considered duplicates. You can have the same name in different stages or choose different names for multiple instances of the same action inside the same stage. This is needed to provide process resuming, where the (stage name, action name) pair is used as an unique key.

Reply:
Ok then. However it could complicate maintenance, when actions added by some specific command line options.